scripts/rnnlm/get_special_symbol_opts.py

```diff
@@ -42,13 +42,19 @@
         if special_symbols[sym][0] != lower_ids[sym]:
             print('{0}={1} '.format(special_symbols[sym][1], lower_ids[sym]), end='')
             printed = True
+        elif special_symbols[sym][0] == lower_ids[sym]:
```

Comment:
This doesn't look right.  If you don't want this to be conditional, there is no need for an if-statement at all.
What problem were you trying to solve here, exactly?